site/content/en/docs/drivers/hyperkit.md

```diff
@@ -29,6 +29,35 @@ If you are using `dnsmasq` and `minikube` fails, add `listen-address=192.168.64.
 
 If you are running other DNS servers, shut them off or specify an alternative bind address.
 
+### Hyperkit upgrade version
+The version of correct Hyperkit not being in the path could cause issues in starting minikube.
+Try upgrading Hyperkit and ensure the upgraded version is in the path.
+
+The latest version of Hyperkit could be found [here](https://github.com/moby/hyperkit/releases).
+
+You can check your current version by:
+
+```
+hyperkit -v
+```
+
+To upgrade try:
+```
+brew upgrade hyperkit
+```
+
+To verify your hyperkit version please run:
+```
+hyperkit -v
+```
+
+if the version didn't change to the latest version [here](https://github.com/moby/hyperkit/releases), you can check if there is an existing hyperkit in the path.
+```
+which hyperkit
+```
+
+alternatively you might want to install the latest version of docker-desktop.
```

Comment:
I strongly recommend folks try this method first, as it would reduce the possibility of conflicting versions of hyperkit on a machine (though, this so far has not been a problem)

Reply:
sounds good I will  move it up the list.